__tests__/app.test.js

```diff
@@ -107,5 +107,51 @@ describe("/api", () => {
                 })
             })
         })
+        describe("/:article_id", () => {
+            describe("/comments", () => {
+                describe("GET", () => {
+                    test("200: responds with an array of comments for the given article_id", () => {
+                        return request(app)
+                        .get("/api/articles/1/comments")
+                        .expect(200)
+                        .then(({ body: {comments}}) => {
+                            expect(comments).toBeSortedBy("created_at", {descending: true})
+                            comments.forEach((comment) => {
+                                expect(typeof comment.comment_id).toBe("number")
+                                expect(typeof comment.votes).toBe("number")
+                                expect(typeof comment.created_at).toBe("string")
+                                expect(typeof comment.author).toBe("string")
+                                expect(typeof comment.body).toBe("string")
+                                expect(typeof comment.article_id).toBe("number")
```

Comment:
I would check that the article_id is 1 - proving you have the correct comments for the article you have selected in the URL